src/js/chrome/entry.js

```diff
@@ -109,7 +109,7 @@ export function bootstrap(libjwt, initFunc, getUser) {
         login: () => libjwt.jwt.login(),
       },
       isProd: window.location.host === 'cloud.redhat.com',
-      isBeta: () => (window.location.pathname.split('/')[1] === 'beta' ? true : false),
+      isBeta: isBeta,
```

Comment:
No need to assign to same name.
```suggestion
      isBeta,
```